vlanBridge.go

```diff
@@ -436,3 +442,36 @@ func (vl *VlanBridge) processArp(pkt protocol.Ethernet, inPort uint32) {
 		}
 	}
 }
+
+// SendGARP sends GARP for the specified IP, MAC
+func (vl *VlanBridge) SendGARP(ip net.IP, mac net.HardwareAddr, vlanID uint16) error {
+	garpPkt, _ := protocol.NewARP(protocol.Type_Request)
```

Comment:
@jojimt @shaleman : I did think about placing it there. The reason I ended up having a separate API is that from my understanding of the libnetwork design an endpoint could be detached from a sandbox and attached to another sandbox. In that case, could there be any changes in IP, MAC? If not, I could move this to AddLocalEndpoint.